Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec puma -C config/puma.rb
```

Comment:
I think maybe is because you are using Windows, but files expect to have a new line at the end